Makefile

```diff
@@ -13,8 +13,5 @@ test:
 examples:
 	@mkdir -p /tmp/examples && for i in $(echo examples/s3/*); do go build -o /tmp/examples/$(basename ${i:0:-3}) ${i}; done
 
-docs:
```

Comment:
We should change in `checks: vet test examples docs functional-test` here as well

Reply:
done.